examples/tests/src/rearrange.test.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from "fs";
+import * as mupdf from "mupdf";
+import path from "path";
+import { afterEach, beforeEach, describe, expect, it } from "vitest";
+
+const scriptdir = path.resolve(__dirname);
+const filename = path.join(scriptdir, "..", "test.pdf");
+const data = fs.readFileSync(filename);
+
+describe("mupdfjs metadata tests", () => {
+  let document: mupdf.PDFDocument;
+
+  beforeEach(async () => {
+    document = mupdf.Document.openDocument(
+      data,
+      "application/pdf"
+    ) as mupdf.PDFDocument;
+  });
+
+  afterEach(() => {
+    document.destroy();
+  });
+
+  it("should rearrange all pages in document", () => {
+    document.rearrangePages([2, 1, 0]);
+
+    expect(document.countPages()).toBe(3);
+    expect(document.loadPage(0)).toBeInstanceOf(mupdf.PDFPage);
+    expect(document.loadPage(1)).toBeInstanceOf(mupdf.PDFPage);
+    expect(document.loadPage(2)).toBeInstanceOf(mupdf.PDFPage);
+  });
```

Comment:
What about checking against the content of the page to make sure the content has been replaced? What do you think? For example, some text, etc.

Reply:
I am now comparing the JSON return value of toStructuredText, is that what you meant?

Reply:
That's good.